controller/fruitsController.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+
+const router = express.Router();
+// eslint-disable-next-line import/extensions
+const service = require("../services/fruitService.js");
+
+function calculatePrice(price) {
+  const date = new Date();
```

Comment:
duplicate logic getPrice  /  calculatePrice in fruitOperations and  fruitController

Reply:
Resolved.